### PR TITLE
Simplify Test_AddComputeTags.cpp

### PR DESCRIPTION
**Pull request by nilsvu**
The action justs adds a compute tag, so it's enough to test that
the compute tag is indeed added. No need to depend on Evolution
or involve randomized tensors.

This more-than-halves the length of this test file.

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Comment by carmaza** [comment]
The reason to add a compute tag is to actually use it, so I don't see a reason why an expected usage shouldn't be part of a test.

That said, I'm fairly new to `Actions` and the infrastructure to test them so I'll leave the review process to the @sxs-collaboration/spectre-core-devs .

**Review by wthrowe — DISMISSED** [review]
I agree that the old test was overkill.  Checking the intended use is good, but using a simple such use is fine as long as it is expected that the simple case will detect any bugs that would affect more complicated cases.  (If this expectation turns out to be incorrect we will have to add more interesting test cases.)

I think the new test is fine.  Personally I would have actually called the added compute item, but I can't think of any plausible way that that could go wrong without triggering the check here.

**Comment by nilsvu** [comment]
Rebased so that Travis runs tests

**Comment by kidder** [comment]
@wthrowe this was rebased

**Review by wthrowe — APPROVED** [review]

**Comment by nilsvu** [comment]
@carmaza could you approve or review since you wrote the code?

**Comment by carmaza** [comment]
Looks good to me. As said before, I'll let the experts approve this PR.

**Review by kidder — APPROVED** [review]

**Pull request merged**
